lib/chefspec/coverage.rb

```diff
@@ -67,7 +69,26 @@ def add_filter(filter = nil, &block)
 
       true
     end
-
+    #
+    # Change the template for reporting of converage analysis.
+    #
+    # @param [string] path
+    #   The template file to use for the output of the report
+    #
+    # @return [true]
+    #
+    def set_template(file = 'human.erb')
+      [
+        ChefSpec.root.join('templates', 'coverage', file),
+        File.join(Dir.pwd,file)
+      ].each do |template|
+        if File.exist? template
+          @erb = Erubis::Eruby.new(File.read(template))
+          return
+        end
+      end
+      raise ArgumentError, "template specified cannot be found: #{file}"
```

Comment:
If you take a look at other examples, ChefSpec has a way of defining errors that we should copy here